packages/system-api/README.md

```diff
@@ -54,3 +54,19 @@ loadApplication().then(() => {
 | `extendSession`     |           | boolean          | true    | If true, this method call will extend current  user session  |
 | `handler`           |           | Function         |         | Success handler. Called with method's returned value, the same as the Promise resolution value |
 | `errorHandler`      |           | Function         |         | Error handler. The same as the Promise rejection value|
+
+### showNotification()
+ Shows the system notification notification (at the top of the application page)
+ Returns a `Promise` that is resolved with `undefined`.
```

Comment:
Hm. As I understand according to `systemCall` typings it returns `SystemMessage`.

Reply:
As I see, `SystemMessage` has not been described anywhere. I have to describe It or dismiss this type. I think that we have not to describe it, because the `SystemMessage` instance not needed anywhere.